plugins/dev-tools/src/playground/index.js

```diff
@@ -550,12 +595,11 @@ function registerEditorCommands(editor, playground) {
   document.addEventListener('keydown', (e) => {
     const ctrlCmd = e.metaKey || e.ctrlKey;
     if (ctrlCmd && e.keyCode === Blockly.utils.KeyCodes.S) {
-      // Save.
       save();
       e.preventDefault();
     } else if (ctrlCmd && e.keyCode === Blockly.utils.KeyCodes.ENTER) {
-      // Load.
-      load();
+      loadXml();
+      loadJson();
```

Comment:
Wouldn't this load more than the desired amount of blocks on the workspace?

Reply:
I don't think so b/c they're both looking at the current value in the editor tab, and only one of them will work correctly based on that (the other will error and return). However, this is confusing, so I added return values to make it more explicit.